bokehjs/src/coffee/models/transforms/jitter.coffee

```diff
@@ -0,0 +1,49 @@
+_ = require "underscore"
+Transform = require "./transform"
+p = require "../../core/properties"
+
+class Jitter extends Transform.Model
+  initialize: (attrs, options) ->
+    super(attrs, options)
+
+  props: ->
+    return _.extend {}, super(), {
+      mean:         [ p.Number, 0        ]
+      width:        [ p.Number, 1        ]
+      distribution: [ p.String, 'uniform']
+    }
+
+  # http://www2.econ.osaka-u.ac.jp/~tanizaki/class/2013/econome3/13.pdf (Page 432)
+  rnorm: (mu, sigma) ->
+    # Generate a random normal with a mean of 0 and a sigma of 1
+    r1 = null
+    r2 = null
+    loop
+      r1 = Math.random()
+      r2 = Math.random()
+      r2 = (2*r2-1)*Math.sqrt(2*(1/Math.E))
+      break if -4*r1*r1*Math.log(r1) >= r2*r2
+    rn = r2/r1
+
+    # Transform the standard normal to meet the characteristics that we want (mu, sigma)
+    rn = mu + sigma*rn
+
+    return rn
```

Comment:
I shifted this over and imported the math.coffee code in as math.  Not sure if that is a good idea since it is so close to "Math" which is the core JS math routines.  I think I am going to rename that bokeh_math.